Marlin/src/lcd/menu/menu_probe_offset.cpp

```diff
@@ -133,11 +133,13 @@ void prepare_for_probe_offset_wizard() {
 
     // Probe for Z reference
     ui.wait_for_move = true;
-    z_offset_ref = probe.probe_at_point(wizard_pos, PROBE_PT_RAISE, 0, true);
+    z_offset_ref = probe.probe_at_point(wizard_pos, PROBE_PT_STOW, 0, true);
```

Comment:
PullRequest https://github.com/MarlinFirmware/Marlin/pull/20344 was in draft when merged. So i think it was not tested.

The Idea of PROBE_PT_RAISE was, that there should be a z rise after probing. I didn't realize that PROBE_PT_RAISE excludes a STOW (but it's logic 🙃 ).

I think we should add the following after probing:
```
// Rise Nozzle after probing for probes, they don't rise when stowing.
#if EITHER(FIX_MOUNTED_PROBE, NOZZLE_AS_PROBE)
  do_z_raise(_MAX(Z_CLEARANCE_BETWEEN_PROBES, Z_CLEARANCE_DEPLOY_PROBE));
#endif
```

Reply:
Too bad there is no `PROBE_PT_RAISE_STOW` as a combination.

I wonder what the tradeoff is in terms of code bytes to create that in `probe.cpp` (couple of bytes only I think). But that code increase would hit _everyone_ that has a probe, and in here it only hits those that activate this functionality. Your thoughts?

Also I don't really want to mess around with `probe.cpp` any more in this lifetime.

Reply:
`Z_CLEARANCE_DEPLOY_PROBE` will be used along with `PROBE_PT_RAISE_STOW`.

Other commands that probe multiple times/points only stow the probe at the end of the entire sequence. They only raise between points. They then provide an argument such as `G29 E` if your probe needs to stow after each probe. This is useful for probes such as solenoids which may overheat if deployed for too long.

Some probes require a slow or manual deploy/stow process, so you don't want to actually deploy/stow multiple times.

Reply:
That is an ideal though. This PR doesn't have to wait for a larger rework.

Reply:
Nevermind, I was confusing this with G35. Always looking a code out of context 😉

Reply:
> I think we should add the following after probing:

@swissnorp I think that capability could make sense. I don't know if it belongs as part of the probe code itself or simply as a move at the end of the wizard. I can imagine there will be someone annoyed and reporting bugs if it were made the behavior after every probe.

Reply:
> 
> 
> Too bad there is no `PROBE_PT_RAISE_STOW` as a combination.
> 
> I wonder what the tradeoff is in terms of code bytes to create that in `probe.cpp` (couple of bytes only I think). But that code increase would hit _everyone_ that has a probe, and in here it only hits those that activate this functionality. Your thoughts?
> 
> Also I don't really want to mess around with `probe.cpp` any more in this lifetime.

@FanDjango I was thinking about the same even before you had posted it. 😄 
But I finaly think its not needed.
I realy like the solution you provided in https://github.com/MarlinFirmware/Marlin/pull/20439



> 
> 
> > I think we should add the following after probing:
> 
> @swissnorp I think that capability could make sense. I don't know if it belongs as part of the probe code itself or simply as a move at the end of the wizard. I can imagine there will be someone annoyed and reporting bugs if it were made the behavior after every probe.

@sjasonsmith I didn't realize that it was a private function and do_z_clearance would be a better solution because do_z_raise does lift the nozzle even higher with a negative `probe_offset.z`. But I think https://github.com/MarlinFirmware/Marlin/pull/20439 is the way to go!
https://github.com/MarlinFirmware/Marlin/blob/e8177735a00d1d7bb20b439544d3324fe01cfe7e/Marlin/src/module/probe.cpp#L259-L267